src/worker_manager/ancillary_files.py

```diff
@@ -280,23 +281,32 @@ def _update_custom_yml_parameters(non_tiled_city_data, updated_aoi):
     return list_doc
 
 
+def add_tile_name_column(tile_grid):
+    tile_grid['tile_name'] = (tile_grid.index
```

Comment:
Are you using `tile_name` later when naming the tiles? I thought the folders were numbered, not the tiles?